contracts/pods/PodAdminGateway.sol

```diff
@@ -190,6 +209,41 @@ contract PodAdminGateway is CoreRef, IPodAdminGateway {
         }
     }
 
+    /// @notice Admin functionality to turn off pod membership transfer
+    /// @dev Permissioned to GOVERNOR, POD_ADMIN, GUARDIAN and the specific role
+    function lockMembershipTransfers(uint256 _podId)
```

Comment:
Are all membership transfers locked by default?

Reply:
Not by default, this switch needs to be toggled on by the admin and at the moment requires it the pod operator creating another pod to disable transfers. Will look at disabling by default

Reply:
agree it should be disabled by default. The factory can have the role and atomically disable

Reply:
Yep agreed